pyproject.toml

```diff
@@ -66,6 +66,16 @@ exclude = [
   'src/scicookie/{{cookiecutter.project_slug}}/*'
 ]
 
+[tool.ruff]
+max-line-length = 79
+exclude = [
+  '.git/*',
+  '__pycache__/*',
+  'docs/*',
+  'src/scicookie/hooks/*',
+  'src/scicookie/{{cookiecutter.project_slug}}/*'
```

Comment:
Hmm, why ignore the template package?

Reply:
because the jinja template .. otherwise it will raise a bunch of error